package.json

```diff
@@ -74,6 +74,7 @@
     "gettext-parser": "1.3.0",
     "glob": "~4.0.5",
     "gulp": "^3.9.1",
+    "husky": "^1.3.1",
```

Comment:
Lets keep using `npm` scripts

Reply:
So, husky lets us hook `npm` scripts to git hooks - so we automate the build on each branch checkout (since we're removing the compiled files, we really need to do it every time to be sure we have the correct files in place)